packages/pulumi-aws/src/utils/uploadFolderToS3.ts

```diff
@@ -139,7 +139,7 @@ export const uploadFolderToS3 = async ({
                                 ...fields,
                                 "Content-Type": contentType || "",
                                 "X-Amz-Meta-Checksum": checksum,
-                                file: fs.readFileSync(path, "utf8")
+                                file: fs.readFileSync(path)
```

Comment:
@chiizzy1 you'll need to adapt the TS type for `data`, because of this error here:
https://github.com/webiny/webiny-js/pull/2635/checks#step:12:27

Previously, all values were simple strings, but with this change, `file` is now a Buffer, and not a string.

Reply:
Hi @Pavel910, I don't know much react, but do you think either of these could work?
- file: JSON.parse(fs.readFileSync(path, 'utf-8'))
or this
-  file: fs.readFileSync(path.toString())

Reply:
@chiizzy1 

Just add the `| Buffer` to the `data` const's type:

```ts
const data: Record<string, string | Buffer> = {
```

This could probably be done a bit smarter / cleaner, but in order to simplify things and since this is an internal / narrow-scoped thing, this will be fine for now.

Reply:
@adrians5j Thanks for the help.

Reply:
@Pavel910 please review my PR.